packages/default/scss/datetime/_theme.scss

```diff
@@ -37,7 +37,18 @@
                         $datetime-select-hovered-border,
                         $datetime-select-hovered-gradient
                     );
+
+                    .k-link:hover {
```

Comment:
We do not need this nesting.
The `.k-link:hover` styles should be moved one level up, so they are not nested in:

```
            // Hovered state
            &:hover,
            &.k-state-hover
```